### PR TITLE
CC 18, test doesn't pass but the comparison is correct, couldnt resol…

### PR DESCRIPTION
**Pull request by Navelfuzz**
…ve issues

### CODE REVIEW

**Comment by Navelfuzz** [comment]
- [x] Top-level README “Table of Contents” is updated
 - [x] README for this challenge is complete
       - [x] Summary, Description, Approach & Efficiency, Solution
       - [ ] Picture of whiteboard
       - [x] Link to code
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
       - [x] “Happy Path” - Expected outcome
       - [x] Expected failure
       - [x] Edge Case (if applicable/obvious)

File: java/datastructures/lib/src/test/java/datastructures/trees/TreesTest.java [hunk 1]
```diff
@@ -56,7 +54,24 @@ public void testBreadthFirstTraversal() {
...
+        binaryTree.root.left = new Node(3);
+        binaryTree.root.right = new Node(10);
+        binaryTree.root.left.left = new Node(9);
+        binaryTree.root.left.right = new Node(5);
+        binaryTree.root.right.left = new Node(7);
+        binaryTree.root.right.right = new Node(11);
+
+        BinaryTree fizzBuzzTree = binaryTree.fizzBuzzTree();
+        List<Object> expected = List.of("FizzBuzz", "Fizz", "Buzz", "Fizz", "Buzz", 7, 11);
+        assertEquals(expected, fizzBuzzTree.breadthFirst());
```
Comment (lucy-gelderloos) [inline java/datastructures/lib/src/test/java/datastructures/trees/TreesTest.java:74]: `expected` will never equal `fizzBuzzTree.breadthFirst()` because they are two different objects. Instead, you need to compare the values in `expected` to the values in `fizzBuzzTree.breadthFirst()`. One way to do that is to convert them both to strings & compare the strings
